parsec/stubs/marshmallow/__init__.pyi

```diff
@@ -2,26 +2,26 @@ from __future__ import annotations
 
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
 
-from typing import Dict, Callable, TypeVar, Any
+from typing import Dict, Callable, Generic, Iterable, TypeVar, Any
 from .fields import Field
 
 class ValidationError(Exception):
     @property
     def messages(self) -> list[str] | str | dict[str, str]: ...
 
-class MarshalResult:
-    def __init__(self, result_data, result_errors) -> None: ...
+class MarshalResult(Generic[T]):
+    def __init__(self, result_data: object, result_errors: object) -> None: ...
```

Comment:
I can't put `list[str]` here because there's some places where a `dict` is passed here (https://github.com/Scille/parsec-cloud/blob/master/parsec/serde/schema.py#L140-L148)